_extras/checkout.md

```diff
@@ -49,40 +49,52 @@ and must be requested before the three months are up. To request an extension, p
 ## Part 1: Submit a Small Contribution to One of Our Lessons
 <a name="lesson-change"></a>
 
-Our lessons are maintained and improved by the people who teach them,
-so this part of the checkout procedure gives you a chance to familiarize yourself
+Our lessons and lesson infrastructure are maintained and improved by the people who use them.
+With this part of the checkout procedure, you will familiarize yourself
 with our collaborative curriculum development process.
 
-Trainees must make a contribution to a lesson's content, exercises, or instructor's guide by doing one of the following:
+### Examples of Contributions
 
-1. Providing substantive feedback on an existing issue or pull request (preferred).
-2. Submitting a change to fix an existing issue.
-3. Proof-reading a lesson and adding a new issue describing something to be improved.
+Trainees must make a contribution to any GitHub repository in The Carpentries organization that is actively maintained and supports a lesson (e.g. [LC Open Refine](https://github.com/LibraryCarpentry/lc-open-refine/), [SWC Unix Shell](http://swcarpentry.github.io/shell-novice/)) or lesson infrastructure (e.g. the [handbook](https://github.com/carpentries/handbook), [workshop template](https://github.com/carpentries/workshop-template)). 
```

Comment:
Does naming specific lessons help here? I'm concerned it means that these (might) get a disproportionate number of contributions at the expense of other lessons.

Reply:
This seems incorrect in the language of "and supports a lesson" as per https://github.com/carpentries/handbook/issues/489 ? Raising an issue in slack about it.

Reply:
How about linking to [GitHub.com/Carpentries](https://github.com/carpentries/), [/DataCarpentry](https://github.com/datacarpentry/), [/SwCarpentry](https://github.com/swcarpentry/) and [/LibraryCarpentry](https://github.com/LibraryCarpentry/) instead. On those org overview pages, issues that "need help" are highlighted.